### PR TITLE
WIP: The Doc's Update #1

### PR DESCRIPTION
**Pull request by moelrobi**
The PR for Issue #1

### CODE REVIEW

**Comment by coffeeboo** [comment]
If you didn't actually make any docs yet, I'll put my Jekyll page in (although it is incomplete)

**Comment by moelrobi** [comment]
Well i had automated Generated Docs. (See https://docs.robsys.space/dev) but i am not Happy with them.

Would like to see your type of Docs. ^^

**Comment by coffeeboo** [comment]
I think I'm creating a new branch for it, you can check it out

**Comment by moelrobi** [comment]
This PR is praticly finished.

After the Docs Merge that will be done by @coffeeboo Version 0.1 can be published 😄 👍

File: index.js [hunk 1]
```diff
@@ -1,158 +1,195 @@
...
+ * @returns {Object} Object where all the Messages in the requested Chat are contained in an Array.
+ */
+async function getChat(sid, com, uid, count) {
+    let msgList = objs.recivedMessages;
+    if(typeof sid != 'string' || typeof com !== 'string' || typeof uid !== 'string') {
+        throw new Error('All Arguments are not satisfied.');
+    }
+    if(count == undefined || count == null) {
+        count = 1;
+        console.log('[AminoJS.getChat()] No count Variable Defined. Defaulting to 1'); // eslint-disable-line
```
Comment (moelrobi) [inline index.js]: General Question. Should this Stay here?

Comment (coffeeboo) [reply index.js]: A silent fallback is more favorable.

Comment (moelrobi) [reply index.js]: will do.

**Comment by moelrobi** [comment]
@coffeeboo and @akatsukilevi could you two review the PR? ^^

**Pull request merged**